src/Components/App/App.js

```diff
@@ -29,12 +44,19 @@ class App extends Component {
       <div className='App'>
         <Header />
 
-        {!this.state.currentMovie && (
+        {this.state.isLoading && !this.state.error &&
+        ( <h2 className="userMsg">Loading...</h2> )}
+
+        {this.state.error && (
+          <h2 className="userMsg">{this.state.error}</h2>
+        )}
+
+        {!this.state.isLoading && !this.state.currentMovie && (
         <Movies movies={this.state.movies} handleClick={this.handleClick}/>
```

Comment:
lots of this.state here - maybe we could destructure isLoading, error, currentmovie etc to save us some keystrokes?